### PR TITLE
F/account scope examples

### PR DESCRIPTION
**Pull request by spasiu**
Make the docs pro account scope JWT

### CODE REVIEW

File: slate/source/includes/rest/core/_authentication.md [hunk 1]
```diff
@@ -93,23 +62,124 @@ The JWT header must contain the key id (`kid`) of the secret key that is used to
...
+
+```javascript
+const jwt = require('jsonwebtoken');
+
+const KEY_ID = 'app_596dead8c82a8c2b00cf0db4';
+const SECRET = 'RiYfZscraDLq1zrXgBdBxU_Z';
+
+const signJwt = function() {
+    return jwt.sign({
+        scope: 'app'
```
Comment (alavers) [inline slate/source/includes/rest/core/_authentication.md]: shouldn't this code sample be appUser scope?

File: slate/source/includes/rest/core/_authentication.md [hunk 2]
```diff
@@ -93,23 +62,124 @@ The JWT header must contain the key id (`kid`) of the secret key that is used to
...
+```javascript
+const jwt = require('jsonwebtoken');
+
+const KEY_ID = 'app_596dead8c82a8c2b00cf0db4';
+const SECRET = 'RiYfZscraDLq1zrXgBdBxU_Z';
+const USER_ID = '08399dbd910734ec28eea677';
+
+const signJwt = function() {
+    return jwt.sign({
+        scope: 'appUser',
```
Comment (alavers) [inline slate/source/includes/rest/core/_authentication.md]: shouldn't this code sample be account scope?

Comment (spasiu) [inline slate/source/includes/rest/core/_authentication.md]: nah, the right-hand code sample goes above the left-hand text...

File: slate/source/includes/rest/core/_authentication.md (hunk 1 above)
Comment (spasiu) [inline slate/source/includes/rest/core/_authentication.md]: as above...

Comment (alavers) [inline slate/source/includes/rest/core/_authentication.md]: 😕  I'll take your word for it :)

**Comment by spasiu** [comment]
Working on updating Swagger based API libs to allow account scope.

**Comment by jpjoyal** [comment]
@spasiu I believe we've missed the webhook APIs 

https://docs.smooch.io/rest/#get-webhook

**Pull request merged**